rom/dev/src/main.rs

```diff
@@ -54,6 +55,13 @@ Running Caliptra ROM ...
 pub extern "C" fn rom_entry() -> ! {
     cprintln!("{}", BANNER);
 
+    if !cfg!(feature = "no-cfi") {
+        cprintln!("[state] CFI Enabled");
+        CfiCounter::reset();
```

Comment:
CfiCounter::reset() inidrectly calls Trng::assume_init(), which is a bug (you can't seed from a trng that hasn't been in initialized yet).

Reply:
Fixed.